src/main/java/gui/MainApp.java

```diff
@@ -74,11 +75,12 @@ private void initLogic() {
     }
 
     private void addCommandBar(MainApp mainApp) {
-        rootLayout.setBottom(new CommandBarController(mainApp));
+    	commandBarController = new CommandBarController(mainApp);
```

Comment:
What's whitespace formatting? Is it a different problem from tab indentation?

Also do you indent empty lines?